nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -296,7 +296,7 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
             'sample_rate': self.preprocessor._sample_rate,
             'batch_size': batch_size,
             'shuffle': False,
-            'num_workers': min(batch_size, os.cpu_count() - 1),
+            'num_workers': min(batch_size - 1, os.cpu_count() - 1),
```

Comment:
@titu1994, I'm getting multiprocessing errors for single file transcribe calls otherwise